include/seqan3/range/views/minimiser.hpp

```diff
@@ -407,26 +507,64 @@ struct minimiser_fn
         return adaptor_from_functor{*this, window_values_size};
     }
 
-    /*!\brief                           Call the view's constructor with two arguments: the underlying view and an
-     *                                  integer indicating how many values one window contains.
-     * \param[in] urange                The input range to process. Must model std::ranges::viewable_range and
-     *                                  std::ranges::forward_range.
-     * \param[in] window_values_size    The number of values in one window.
-     * \returns                         A range of converted values.
+    /*!\brief Call the view's constructor with two arguments: the underlying view and an integer indicating how many
+     *        values one window contains.
```

Comment:
I am not sure, if this description is better, because anybody who does not look to closely into this will assume that this is the same as the window size in minimiser_hash and it is not. Especially, if a user has a window on a DNA text in mind this could lead to confusion.

Reply:
I had the discussion with @smehringer at #1654 . The reason why window_size != window_values_size:
(Quoting myself from #1654)
A window_size of 10 means w - k + 1 elements that are completely in the window, we can also say we want to consider all k-mers starting in a window (which is done in some papers), but in all our example figures we do the former... But this view should be capable of handling both, it depends on what an user defines for window_values_size, so I still would say not naming it window_size is a good idea.

Reply:
I am confused? This view says: Give me the minimiser in the window move the window and give me the next minimiser. So whatever other views do, in this case I give it a window of size 10 and it finds the minimiser over this size or not?